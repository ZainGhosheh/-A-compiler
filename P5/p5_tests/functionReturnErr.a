f : () -> void {
	a:int;
	a = 4;
    return a;
}
