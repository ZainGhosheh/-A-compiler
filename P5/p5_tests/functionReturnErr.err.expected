FATAL [4,5]-[4,13]: Return with a value in void function
Type Analysis Failed
